ltac/tacinterp.ml

```diff
@@ -1422,7 +1422,9 @@ and tactic_of_value ist vle =
         extra = TacStore.set ist.extra f_trace []; } in
       let tac = name_if_glob appl (eval_tactic ist t) in
       Profile_ltac.do_profile "tactic_of_value" trace (catch_error_tac trace tac)
-  | (VFun _|VRec _) -> Tacticals.New.tclZEROMSG (str "A fully applied tactic is expected.")
+  | VFun (_, _, _,vars,_) ->
+    Tacticals.New.tclZEROMSG (str "A fully applied tactic is expected: missing " ++ pluralize "argument" vars ++ Pp.str " " ++ pr_name_list vars)
```

Comment:
I don't know the opinion of others but I feel that calling an argument `_` in the message "A fully applied tactic is expected: missing arguments y _" is not very crystal clear. May I suggest "A fully applied tactic is expected: missing arguments for variables y _." since after all `_` is not an argument but really a variable.

By the way, there is already a printer for sequences. It is called `Pp.pr_sequence`. I feel that using it would give an even more natural message, namely: "A fully applied tactic is expected: missing arguments for variables y and _."

Also, the final "." is missing in the message.

In any case, thanks for your useful initiative!